sync/class.jetpack-sync-module-users.php

```diff
@@ -84,6 +91,14 @@ private function get_user( $user ) {
 		return null;
 	}
 
+	public function add_invitation_accepted_flag( $user ) {
+		if ( empty( $user ) ) {
+			$this->invitation_accepted = true;
```

Comment:
why not use `add_flags` directly here?

Reply:
The user hasn't been created yet. :( We could use the email address though